assignments/function-conversion.js

```diff
@@ -5,26 +5,48 @@
 // };
 // myFunction();
 
+let myFunction = () => {
+console.log("Function was invoked!");
+};
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => { return param };
```

Comment:
Great work with arrow functions. 
You could clean your syntax here even more in this fashion:
`let anotherFunction = param => param;`

Notice how when you declare your function with just 1 param you can just skip parenthesis in your declaration.
Also if the body of your function is just a return value you can skip curly brackets and the return keyword.